c/parallel/include/cccl/c/for.h

```diff
@@ -21,14 +21,14 @@
 
 CCCL_C_EXTERN_C_BEGIN
 
-struct cccl_device_for_build_result_t
+typedef struct cccl_device_for_build_result_t
```

Comment:
No action needed. If we have a typedef, technically the struct can be unnamed and our public APIs would not care:
```suggestion
typedef struct
```